problem2/problem2.rb

```diff
@@ -0,0 +1,23 @@
+class Problem2
+	attr_accessor :result,:array
+
+	def  initialize(*args)
+		@array = args;
+	end
+
+	def Largest2Numbers
+		@result=[0,0];
+		for i in 0..(@array.size-1)
+			if(!@result.include? @array[i])
```

Comment:
This is not a great check, `include?` is O(n) so your implementation now is O(n^2), do you really need it?